Assets/MRTK/Examples/Experimental/ExamplesHub/Profiles/MRTKExamplesHubSceneSystemProfile.asset

```diff
@@ -16,7 +16,7 @@ MonoBehaviour:
   useManagerScene: 0
   managerScene:
     Name: DefaultManagerScene
-    Path: Assets/MRTK/SceneSystemResources/DefaultManagerScene.unity
+    Path: Assets/MRTK/Services/SceneSystem/SceneSystemResources/DefaultManagerScene.unity
```

Comment:
Is this change intended for this PR?

Reply:
this looks like it got missed from a different PR? this is the current location for the files.

Reply:
maybe it's GH being confused?

Reply:
Hrm yeah, strange. It's in this commit for some reason https://github.com/microsoft/MixedRealityToolkit-Unity/pull/8786/commits/8cc16e7e3edb13f1742e141e76779ba65142ee1c

Reply:
That is odd... It _is_ the correct path, however.

Reply:
provided that nothing else changed in the profile (index, order, etc). this should be what we want.